exercises/advriskmin/ex_rnw/sol_ex_advriskmin_2_22.Rnw

```diff
@@ -0,0 +1,218 @@
+	1, & \mbox{if ($\pix\geq 1/2$ \& $y=0$) or ($\pix< 1/2$ \& $y=1$), } \\
+	0, & \mbox{else. }
+\end{cases}  } \right) .$$
+%
+We use our usual ``unraveling trick'' by means of the law of total expectation:
+%
+\begin{align*}
+%	
+	\Exy\left[  L(y,\pi(\xv)) \right] 
+	%	
+	&=  \E_x \left[ \E_{y|x} [ L(y, \pi(\xv)) ~|~ \xv = \xv ] \right]  
+%	
+\end{align*}
+%
+and consider then minimization of $ \E_{y|x} [ L(y, \pi(\xv)) ~|~ \xv = \xv ]$ by choosing $ \pi$ point-wise, i.e., for any point $\xv$.
+%
+With the alternative form of $L$, we obtain
+%
+\begin{align*}
+%	
+	\E_{y|x} &[ L(y, \pi(\xv)) ~|~ \xv = \xv ]  \\
+%	
+	&= \E_{y|x} [ \mathds{1}_{\{ \pix\geq 1/2  \}}  \mathds{1}_{\{ y=0 \}} + \mathds{1}_{\{ \pix< 1/2  \}} \mathds{1}_{\{ y=1 \}} ~|~ \xv = \xv ] \\
+	%	
+	&= \E_{y|x} [ \mathds{1}_{\{ \pix\geq 1/2  \}}  \mathds{1}_{\{ y=0 \}}  ~|~ \xv = \xv ] \quad +  \E_{y|x} [ \mathds{1}_{\{ \pix< 1/2  \}} \mathds{1}_{\{ y=1 \}} ~|~ \xv = \xv ] \tag{Linearity of expectation} \\
+%	
+	&=  \mathds{1}_{\{ \pix\geq 1/2  \}}  \cdot \E_{y|x} [ \mathds{1}_{\{ y=0 \}}  ~|~ \xv = \xv ] ~ + \mathds{1}_{\{ \pix< 1/2  \}} \cdot \E_{y|x} [ \mathds{1}_{\{ y=1 \}} ~|~ \xv = \xv ] \tag{$\mathds{1}_{\{ \pix\geq 1/2  \}}$ and $\mathds{1}_{\{ \pix < 1/2  \}}$ are non-random given $\xv$ }\\
+%
+	&=  \mathds{1}_{\{ \pix\geq 1/2 \}}  \P(y = 0~|~ \xv = \xv) \qquad ~ ~ + \mathds{1}_{\{ \pix < 1/2 \}} \P(y = 1~|~ \xv = \xv). \tag{Expectation of an indicator random variable} 
+%	
+\end{align*}
+%
+We can distinguish between two cases:
+%
+\begin{itemize}
+%	
+	\item If $ \P(y = 0~|~ \xv = \xv) \geq  \P(y = 1~|~ \xv = \xv) $ (or  $\P(y = 0~|~ \xv = \xv)\geq 1/2$), then any $\pix$ such that $\pix < 1/2$ minimizes $\E_{y|x} [ L(y, \pi(\xv)) ~|~ \xv = \xv ] .$
+	
+%	
+	\item If $ \P(y = 0~|~ \xv = \xv) \leq  \P(y = 1~|~ \xv = \xv) $ (or  $\P(y = 0~|~ \xv = \xv)\leq 1/2$), then any $\pix$ such that $\pix \geq 1/2$ minimizes $\E_{y|x} [ L(y, \pi(\xv)) ~|~ \xv = \xv ] .$
+%	
+\end{itemize}
+%
+Thus, any $\pi$ of the form
+%
+\begin{align} \label{def_minim_prob_class}
+%	
+	\pix 
+%	
+	= \begin{cases}
+			< 1/2 , & \mbox{if $\P(y = 0~|~ \xv = \xv)\geq 1/2,$} \\
+			\geq 1/2 ,    & \mbox{if $\P(y = 0~|~ \xv = \xv)< 1/2,$} 
+	\end{cases}
+%	
+\end{align}
+%
+minimizes $	\Exy\left[  L(y,\pi(\xv)) \right]$ over $\Hspace = \{ \pi:\Xspace \to [0,1]    \}.$
+%
+The posterior distribution $p_{y|x}(1 ~|~ \bm{x})$ is quite naturally of this form, but it is in general not the only $\pi$ of this kind.
+%
+As a consequence, the minimizer is not unique.
+
+\emph{(Dis-)advantages.}  The posterior distribution $p_{y|x}$ is the \emph{ground-truth} we seek to find with our (empirical) loss minimization approach. 
+%
+Thus, the corresponding loss function should give an incentive for any learning algorithm to do this by minimizing the loss function.
```

Comment:
"to do this" = do what?

Reply:
find the ground-truth. Habe ich hinzugefügt